core/domain/rte_component_registry_test.py

```diff
@@ -59,11 +59,14 @@ def _validate_customization_arg_specs(self, customization_arg_specs):
             self.assertTrue(isinstance(ca_spec['description'], basestring))
             self.assertGreater(len(ca_spec['description']), 0)
 
+            #Image component has a required field whose default value is
+            #non-empty, hence a False is passed as argument so that the tests
+            #pass
             schema_utils_test.validate_schema(ca_spec['schema'])
             self.assertEqual(
                 ca_spec['default_value'],
                 schema_utils.normalize_against_schema(
-                    ca_spec['default_value'], ca_spec['schema']))
+                    ca_spec['default_value'], ca_spec['schema'], False))
```

Comment:
When passing in args that have default values, include the name of the arg. I.e. `apply_custom_validators=False` rather than just `False`.